src/hoc/modals/ReadPillModal/ElementTypeRender/index.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import EllipseIcon from '../../../../assets/icons/EllipseIcon';
+import { SquaredIcon } from '../../../../assets/icons/SquaredIcon';
+import {
+  StyledColumn,
+  StyledImage,
+  StyledRow,
+  StyledText,
+} from '../../../../components/styled/styles';
+import { removeHtmlTags } from '../../../../utils/utils';
+
+export interface BubbleProps {
+  type: string;
+  id: string;
+  metadata?: {
+    metadata?: { lerni_question_type: string };
+    options?: string[];
+  };
+  name: string;
+  question_type?: string;
+}
+
+export const ElementTypeRender = ({ type, id, metadata, name, question_type }: BubbleProps) => {
+  if (type === 'ACTION') {
+    if (!metadata) {
+      return (
+        <StyledColumn css={{ justifyContent: 'center', alignItems: 'left', gap: '4px' }}>
+          <StyledText variant="body3" color="primary500" css={{ fontFamily: 'Roboto-Bold' }}>
+            {'Text'}
+          </StyledText>
+          <StyledText variant="body2" color="gray950">
+            {removeHtmlTags(name)}
+          </StyledText>
+        </StyledColumn>
+      );
+    } else if (metadata && metadata.metadata && metadata.metadata.lerni_question_type === 'image') {
+      return (
+        <StyledColumn css={{ justifyContent: 'center', alignItems: 'left', gap: '4px' }}>
+          <StyledText variant="body3" color="primary500" css={{ fontFamily: 'Roboto-Bold' }}>
+            {'Imagen'}
+          </StyledText>
+          <StyledImage src={name} css={{ borderRadius: '8px' }} width={150} height={150} />
```

Comment:
Should images be displayed as squares if their original size isn't one?

Reply:
<img width="551" alt="image" src="https://github.com/sirius-valley/lerni-web/assets/29240378/e623a4ee-ee91-4363-86b0-632e1f7a453d">

<img width="172" alt="image" src="https://github.com/sirius-valley/lerni-web/assets/29240378/3d782ed4-25f0-4ff7-b9a9-2a38dde05c22">